lib/node_modules/@stdlib/stats/base/ndarray/dmidrange/docs/types/index.d.ts

```diff
@@ -0,0 +1,46 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { float64ndarray } from '@stdlib/types/ndarray';
+
+/**
+* Compute the midrange value of a one-dimensional double-precision floating-point ndarray.
```

Comment:
```suggestion
* Compute the mid-range value of a one-dimensional double-precision floating-point ndarray.
```

Use descriptions and spelling which are consistent with other packages (e.g., `stats/strided/dmidrange`).

Reply:
This applies here and everywhere else in this PR.